.github/workflows/release_droid_prepare_original_checksum.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Run tests and build with Maven
         run: mvn --batch-mode clean verify --file pom.xml
       - name: Prepare checksum
-        run: find target -maxdepth 1 -name *.jar -exec sha256sum "{}" + > original_checksum
+        run: find . -regex '.*/target/.*\.jar' -exec sha256sum "{}" + > original_checksum
```

Comment:
Does this also work for single-module projects? If yes we should use the same mechanism in project-keeper.

Reply:
My short experiment looked good. It found './target/<blah>.jar'

Reply:
Created https://github.com/exasol/project-keeper/issues/474 to add this to PK